spec/features/state_file/login_spec.rb

```diff
@@ -52,8 +52,6 @@
     expect(page).to have_text "Code verified! Authentication needed to continue."
     fill_in "Enter your Social Security number or ITIN. For example, 123-45-6789.", with: ssn
     click_on "Continue"
-
-    expect(page).to have_text "Your federal tax return is now transferred."
   end
```

Comment:
[pebble] removing these expectations leaves the test in a weird state. the last line in each of these tests is clicking on "Continue." I think it would be preferable to have an expectation. 

Is there an expectation that can be added here after clicking on continue? Possibly testing that the SSN is being added to the DB?